app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt

```diff
@@ -65,175 +65,200 @@ fun FriendListView(
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            // Image painter for loading from a URL
+                            val imagePainter =
+                                rememberAsyncImagePainter(model = friend.profileImageUrl)
 
-                        Image(
-                            painter = imagePainter,
-                            contentDescription = "${userProfile.username}'s profile picture",
-                            contentScale = ContentScale.Crop,
-                            modifier =
+                            Image(
+                                painter = imagePainter,
+                                contentDescription = "${userProfile.username}'s profile picture",
+                                contentScale = ContentScale.Crop,
+                                modifier =
                                 Modifier.size(62.dp)
                                     .clip(RoundedCornerShape(50))
-                                    .align(Alignment.CenterVertically))
-                        Column(
-                            modifier = Modifier.fillMaxHeight().padding(start = 15.dp),
-                            verticalArrangement = Arrangement.Center) {
-                              Text(
-                                  text = friend.username,
-                                  style =
-                                      TextStyle(
-                                          fontSize = 16.sp,
-                                          lineHeight = 16.sp,
-                                          fontFamily = FontFamily(Font(R.font.montserrat)),
-                                          fontWeight = FontWeight(600),
-                                          color = Color.White,
-                                          textAlign = TextAlign.Left,
-                                          letterSpacing = 0.5.sp),
-                                  overflow = TextOverflow.Ellipsis,
-                                  maxLines = 1)
-                              Row() {
+                                    .align(Alignment.CenterVertically)
+                            )
+                            Column(
+                                modifier = Modifier.fillMaxHeight().padding(start = 15.dp),
+                                verticalArrangement = Arrangement.Center
+                            ) {
                                 Text(
-                                    text = "${friend.name} ${friend.surname}",
+                                    text = friend.username,
                                     style =
+                                    TextStyle(
+                                        fontSize = 16.sp,
+                                        lineHeight = 16.sp,
+                                        fontFamily = FontFamily(Font(R.font.montserrat)),
```

Comment:
I believe that 'fontFamily = Montserrat' works also

Reply:
I didn't know, thanks a lot !